prdoc/pr_5380.prdoc

```diff
@@ -0,0 +1,15 @@
+title: Fix leases with gaps and time slice calculation in MigrateToCoretime
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Agile Coretime storage migration wasn't transferring correctly leases with gaps and was
+      miscalculating time lease period. This patch provides fixes for both issues.
+
+crates:
+  - name: rococo-runtime
+    bump: patch
+  - name: westend-runtime
+    bump: major
```

Comment:
I personally feel this is an overstatement but `SemVer` check is not accepting a `patch` here probably because I am removing a `pub` type.